common/app/views/fragments/atoms/structureddata/recipe.scala.html

```diff
@@ -1,6 +1,8 @@
 @(recipe: model.content.RecipeAtom)(implicit request: RequestHeader)
 @import play.api.libs.json.Json
 @import views.support.{ImgSrc, GoogleStructuredData}
+@import org.joda.time.DateTime
+@import cats.implicits._
```

Comment:
We don't use `cats` or any FP library in `frontend` anymore. We had a long discussion about it. If you want to introduce `cats` please come talk to us (@guardian/dotcom-platform) before doing so.